components/button/button.js

```diff
@@ -16,15 +16,15 @@ function Button({ children, mods = [], size, href, onClick, type, variation, dis
   }
 
   if (size) {
-    mods.push(`size_${size}`);
+    mods.push(`size-${size}`);
```

Comment:
According to BEM naming convention the dash `-` could be used only for breaking words in long names, but not for separation values